contracts/BatchReveal.sol

```diff
@@ -269,6 +371,9 @@ abstract contract BatchReveal is
         view
         returns (bool, uint256)
     {
+        if (!isBatchReveal()) {
```

Comment:
should we revert with an error or return `(0, false)`?

Reply:
if we return `(false, 0)`, it will also revert anyway

https://github.com/traderjoe-xyz/launchpeg/blob/05c144c1df70e7787c4da206a60a89284efd5e43/contracts/BaseLaunchpeg.sol#L529-L533

I would lean towards using a `batchRevealEnabled` modifier though

Reply:
I'm not a big fan of reverting here, this is also used by the view function `hasBatchToReveal` that shouldn't revert but return false